CHANGELOG.md

```diff
@@ -48,6 +48,7 @@ BUG FIXES:
   [GH-379]
 * Fix `vcd_independent_disk` reapply issue, which was seen when optional `bus_sub_type` and `bus_type` wasn't used - [GH-394]
 * Fix `vcd_vapp_network` apply issue, where the property `guest_vlan_allowed` was applied only to the last of multiple networks.
+* Fix `vcd_network_direct` data reading from Org Users. Such users can now read the data from existing direct networks.
```

Comment:
I suggest rephrasing the message to make it clearer and move it to improvements:

```suggestion
* `resource/vcd_network_direct` Org User can now use this resource (previously it was only Sys Admin) - this change makes it possible to get the details of External Network as Org User
```

Reply:
changed

Reply:
About the documentation page, for the resource it is still valid. Org users can't create a direct network. But they can read it.
We have the warning banner in the resource, but not in the data source